packages/sit-onyx/src/styles/index.scss

```
@@ -8,9 +8,11 @@
 @use "fonts.scss";
 @use "breakpoints.scss";
 
-@use "variables/themes/onyx.css";
-@use "variables/light.css";
-@use "variables/dark.css";
+@use "variables/themes/Value.css";
+
+@use "variables/themes/onyx-dark.css";
+@use "variables/themes/onyx-light.css";
```

```suggestion
@use "variables/themes/onyx-light.css";
@use "variables/themes/onyx-dark.css";
```

Didn't we find out in the daily today that light theme must be imported first?

---

I changed the imports in the theming PR.